### PR TITLE
Refractor&responsive docs

### PR DESCRIPTION
**Pull request by AtulHande03a**
- refractored documentation code
- added more details about components
- In docs added class to cards
- In docs, fixed toast and made it responsive
- in docs, updated navigation and added hamburger
- removed most of inline style for carbon-sh code snippet and added classes to it
-

### CODE REVIEW

File: documentation/docs.css [hunk 1]
```diff
@@ -8,6 +8,10 @@
 
 /* grid-container */
 
+html {
+  scroll-behavior: smooth;
```
Comment (horsemaker) [inline documentation/docs.css:12]: SmoothCoreyVidalGIF

**Review by horsemaker — APPROVED** [review]

File: documentation/docs.css [hunk 2]
```diff
@@ -180,4 +197,65 @@ body {
...
+    display: flex;
+  }
+
+  .hamburger-list {
+    position: absolute;
+    top: 3.5rem;
+    right: 1rem;
+    z-index: 10;
+    outline: 1px solid var(--gray);
+    gap: 0.7rem;
```
Comment (shivanigangadharan) [inline documentation/docs.css:256]: Just a suggestion, for less than 20px you can use px as the unit.

**Review by shivanigangadharan — APPROVED** [review]
Code looks good, nice UI. Can merge 👍

File: documentation/docs.css [hunk 3]
```diff
@@ -180,4 +197,65 @@ body {
...
+    background-color: var(--white);
+    cursor: pointer;
+    padding: 0.7rem 0.7rem;
```
Comment (NeerajVodala) [inline documentation/docs.css:259]: why two 0.7's?

**Review by NeerajVodala — APPROVED** [review]
LGTM.

File: documentation/docs.css (hunk 3 above)
Comment (AtulHande03a) [reply documentation/docs.css:259]: nav-ex

so on click of that hamburger , that list opens and 0.7 looked better padding than 1 so used it.

File: documentation/docs.css (hunk 2 above)
Comment (AtulHande03a) [reply documentation/docs.css:256]: wanted to remain consistent with units but will make use of this in future

File: documentation/docs.css (hunk 3 above)
Comment (NeerajVodala) [reply documentation/docs.css:259]: I mean `padding: 0.7rem;` is enough😅

**Pull request merged**